docs/tutorials/get-started/try-viam-sdk.md

```diff
@@ -202,10 +202,100 @@ npm start
 
 Open a web browser and visit `localhost:8000`.
 You should see a disabled button that says `Click me`.
-If you successfully configured your robot and it is able to connect to the Viam app, the button will become enabled.
-If you open the developer console, you should see some output including the names of your rover's resources.
+Open the developer console by right-clicking the page and selecting `Inspect`, then selecting the console tab.
```

Comment:
it's different between different browsers, so I think this may be more generally applicable
```suggestion
Open the developer console to see the console output.
```